examples/invoker/measure.go

```diff
@@ -10,7 +10,8 @@ import (
 	"go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc"
 	"google.golang.org/grpc"
 
-	"eventing/proto"
+	"github.com/ease-lab/vhive/utils/benchmarking/eventing/proto"
+
 	"github.com/ease-lab/vhive/examples/endpoint"
 )
 
```

Comment:
please rename `conn` and `client`, indicating that they correspond to the timeseries db service